infrastructure/prod.tfvars

```diff
@@ -62,3 +62,5 @@ es_instance_count                 = 3
 es_instance_volume_size           = 512
 create_db_accessor_instance       = true
 db_accessor_instance_class        = "t3.2xlarge"
+create_elk_instance               = true
```

Comment:
Good here but also need to add the same two variables to infrastructure/staging.tfvars